.github/workflows/integration-tests.yml

```diff
@@ -13,14 +60,38 @@ on:
         required: true
       FIREBOLT_CLIENT_SECRET_STG_NEW_IDN:
         required: true
+
 jobs:
   integration-test-v1:
+    if: ${{ inputs.run-v1 == 'true' }}
     uses: ./.github/workflows/integration-tests-v1.yml
+    with:
+      database: ${{ inputs.database1 }}
+      engine: ${{ inputs.engine1 }}
     secrets:
       FIREBOLT_STG_USERNAME: ${{ secrets.FIREBOLT_STG_USERNAME }}
       FIREBOLT_STG_PASSWORD: ${{ secrets.FIREBOLT_STG_PASSWORD }}
+
   integration-test-v2:
+    if: ${{ inputs.run-v2 == 'true' }}
+    uses: ./.github/workflows/integration-tests-v2.yml
+    with:
+      database: ${{ inputs.database2 }}
+      engine: ${{ inputs.engine2 }}
+      account: ${{ vars.FIREBOLT_ACCOUNT_V1 }}
+      run: dotnet test --filter 'Category=Integration&(Category=v2|Category=general)' -l "console;verbosity=normal"
+    secrets:
+      FIREBOLT_CLIENT_ID_STG_NEW_IDN: ${{ secrets.FIREBOLT_CLIENT_ID_STG_NEW_IDN }}
+      FIREBOLT_CLIENT_SECRET_STG_NEW_IDN: ${{ secrets.FIREBOLT_CLIENT_SECRET_STG_NEW_IDN }}
+  
+  integration-test-engine-v2:
+    if: ${{ inputs.run-database-with-engine-v2 == 'true' }}
     uses: ./.github/workflows/integration-tests-v2.yml
+    with:
+      database: ${{ inputs.database-with-engine-v2 }}
+      engine: ${{ inputs.engine_v2_fb_2_0 }}
+      account: ${{ vars. FIREBOLT_ACCOUNT_V2 }}
```

Comment:
1. thanks. 
2. the dies not work because it fails on the DB&ENGINE setup stage because the script uses option that is not supported now. We discussed this issue with you a couple of weeks ago.

Reply:
> We discussed this issue with you a couple of weeks ago.
I don't remember such discussion, especially going weeks ago.

However, we've discussed on a daily that `integration-testing-setup` will not work with engines v2. There's no need to modify it since setup is simple - connect to system engine and run CREATE DATABASE, CREATE ENGINE, START ENGINE. This can be done in connectors themelves.

Reply:
surely, we discussed this too. I will do this in next, separate, PR